### PR TITLE
show notification on my bets button when wallet has unredeemed bets

### PR DESCRIPTION
**Pull request by Diogomartf**
Show notification on "my bets" button when wallet has unredeemed bets to make it clear when user has to redeem.

**Preview**
image
image

### CODE REVIEW

File: entities/markets/userBets.ts [hunk 1]
```diff
@@ -6,3 +8,32 @@ export interface UserBets extends UserPosition {
   fpmm: FixedProductMarketMaker;
   condition: Condition;
 }
+
+export class Bets {
```
Comment (berteotti) [inline entities/markets/userBets.ts:12]: File is named UserBets but the class is named Bets, should we call it UserBets too? I think this is related to a user positions, so UserBets makes sense in my mind.

Comment (ElRodrigote) [reply entities/markets/userBets.ts:12]: Yeah agree, and if in the future we add another feature like a ranking or sharing bets, or something like that, we can differentiate one Bet from another with the naming suggested by Leo.

File: app/components/Navbar.tsx [hunk 2]
```diff
@@ -20,9 +41,16 @@ export const Navbar = () => {
...
-              My bets
-            </Button>
+            <div className="relative">
+              <Button variant="pastel" className="space-x-1 text-nowrap">
+                My bets
+              </Button>
+              {hasUnredeemedBets && (
+                <div className="absolute -right-2 -top-1 flex size-5 items-center justify-center rounded-100 border-2 border-surface-surface-bg bg-surface-success-main p-1 text-text-white">
+                  <p className="text-xs font-semibold">{unredeemedBetsNumber}</p>
+                </div>
```
[the remark this answers is not in the document]
Comment (Diogomartf) [reply app/components/Navbar.tsx:51]: I read well, but can try black on dark mode

File: entities/markets/userBets.ts (hunk 1 above)
Comment (Diogomartf) [reply entities/markets/userBets.ts:12]: Yeah, makes sense, I will update it to `UserBets`.

File: app/(main)/my-bets/page.tsx [hunk 3]
```diff
@@ -15,41 +15,28 @@ import { BetsListPanel, BetsListTab } from '@/app/components';
...
-        const marketCondition = new MarketCondition(
-          userPosition.fpmm,
-          position.condition
-        );
-        const canRedeem = marketCondition.canRedeem(outcomeIndex, userPosition.balance);
-        return canRedeem;
-      }) ?? [],
-    [userPositionsComplete]
+    () => userBetsManager.getUnredeemedBets(),
+    [userBetsManager]
```
Comment (ElRodrigote) [inline app/(main)/my-bets/page.tsx:39]: This looks so much more clean and elegant, great work!!

**Review by ElRodrigote — APPROVED** [review]

**Pull request merged**